### PR TITLE
Add create and remove functions

### PR DESCRIPTION
**Pull request by vladanvi99**
In this pull request, I added to my project, functions for creating and removing books.

### CODE REVIEW

File: src/components/books/addNewBook/AddNewBook.js [hunk 1]
```diff
@@ -1,19 +1,42 @@
...
+    e.preventDefault();
+    dispatch(createBook(bookInfo));
+    setBookInfo({ ...bookInfo, name: '' });
+  };
+
+  return (
+    <section className="add-new-book-section">
+      <h2>ADD NEW BOOK</h2>
+      <form onSubmit={(e) => onSubmit(e)}>
+        <input onChange={handleChange('name')} type="text" name="name" placeholder="Book title" value={bookInfo.name} />
```
Comment (bellom) [inline src/components/books/addNewBook/AddNewBook.js:29]: Great job.

We need to add second input for the `Author`

image

File: src/components/books/book/Book.js [hunk 2]
```diff
@@ -8,15 +8,15 @@ import './book.scss';
...
     <div className="card-wrap">
       <div className="start">
         <div className="left">
           <p className="category">{category}</p>
           <h3>{name}</h3>
-          <p className="author">{author}</p>
+          <p className="author">Unknown</p>
           <div className="book-options">
             <button type="button">Comments</button>
             <button type="button">Remove</button>
```
Comment (bellom) [inline src/components/books/book/Book.js]: We need to implement the remove book action in this `Book` component.

image

File: src/components/books/book/Book.js [hunk 3]
```diff
@@ -8,15 +8,15 @@ import './book.scss';
 
 const Book = ({ bookInfo }) => {
   const {
-    name, author, category, progress, chapter,
+    name, category, progress,
```
Comment (bellom) [inline src/components/books/book/Book.js]: By the time you added the author input tag, don't forget to add `author` in this destructuring.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @vladanvi99  👏 , 

### Awesome job so far!! :+1: 

#### You have done wonderfully well setting up linter and good commit messages 🔥, However, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 


# Status: Required Changes! ♻️ 

_1. Kindly check the comments under the review._

_2. Let's make sure the add and remove features are working, currently, they are not working, users should be able to add and remove book_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**You could improve your PR description and commit message, I think you did more than what you wrote, you could have listed the mini-features too, and also try to capitalize your commit messages next time.**

Example: 
image

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/books/addNewBook/AddNewBook.js (hunk 1 above)
Comment (dasileker) [reply src/components/books/addNewBook/AddNewBook.js:29]: - [ ] As the previous TSE mentioned you need a second input for the **`author`**  I only see **`Title`**

lll

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @vladanvi99**, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

- You only need to add the **`Author input`** and the project it will be complete you have done a wonderful work in the **`reducer/book.js`** to pass the action type as  `state`.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by hemantso — APPROVED** [review]
Hi @vladanvi99, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 

Alt Text

Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**